pkg/proxy/proxy/transparent.go

```diff
@@ -108,13 +108,18 @@ func NewTransparentProxyService(p *goproxy.ProxyHttpServer, ca *tls.Certificate,
 	return TransparentProxyService{
 		Proxy:      p,
 		Ca:         ca,
-		NetworkLog: netlog.CaptureActivityLog(p, m),
+		NetworkLog: &netlog.NetworkActivityLog{},
```

Comment:
I think I might prefer this being nil on creation. Thoughts?

Reply:
I think either works here. I just added an empty object here in case the `/summary` gets called with this uninitialized. Seemed nicer to have something like 
```
{
  "HTTPRequests": null
}
```
rather than just having
```
null
```
But I'm ok either way.

Reply:
ah right we just fixed that bug in the first place... Okay yeah I think initializing it sounds good.

Would it be possible to make this a private member though? Exposing the log through a method should avoid exposing the implementation details too much now that we're supporting "resetting".

one bigger change would be to move the aggregation up to this level. CaptureActivityLog could just be responsible for providing a channel of log entries and the summary construct could be populated here. If we want to add logic related to memory usage monitoring, doing so here would be a better fit than in the netlog package.

Reply:
Made networkLog private.